app/models/manageiq/providers/google/inventory.rb

```diff
@@ -8,4 +8,8 @@ class ManageIQ::Providers::Google::Inventory < ManageIQ::Providers::Inventory
   def self.default_manager_name
     "CloudManager"
   end
+
+  def self.parser_classes_for(_ems, _target)
+    [ManageIQ::Providers::Google::Inventory::Parser::CloudManager, ManageIQ::Providers::Google::Inventory::Parser::NetworkManager]
```

Comment:
NOTE we can combine these into one parser class, this just made it simpler to get the specs passing initially

Reply:
Done